xarray/tests/test_interp.py

```diff
@@ -809,9 +921,7 @@ def test_decompose(method: InterpOptions) -> None:
 
 @requires_scipy
 @requires_dask
-@pytest.mark.parametrize(
-    "method", ["linear", "nearest", "zero", "slinear", "quadratic", "cubic"]
-)
+@pytest.mark.parametrize("method", ["linear", "nearest"])
```

Comment:
It's not clear how you're choosing which methods to paramterize over?

Perhaps we should add some global constants that can be used repeatedly

```
INTERP_METHODS_SEPARABLE=["linear", "nearest", "slinear"]
INTERP_METHODS_INSEPARABLE = ...
INTERP_METHODS_SUBSET = ["linear", "cubic"]
```